packages/components/src/drop-zone/style.scss

```diff
@@ -1,3 +1,67 @@
+@keyframes components-drop-zone__fadeIn {
+	0% {
+		opacity: 0;
+	}
+	100% {
+		opacity: 1;
+	}
+}
+
+@keyframes components-drop-zone__fadeOut {
+	0% {
+		opacity: 1;
+	}
+	100% {
+		opacity: 0;
+	}
+}
+
+@keyframes components-drop-zone__scaleIn {
+	0% {
+		opacity: 0;
+		transform: scale(0.9);
+	}
+	50% {
+		opacity: 0;
+		transform: scale(0.9);
+	}
+	100% {
+		opacity: 1;
+		transform: scale(1);
+	}
+}
+
+@keyframes wp-components-drop-zone__scaleOut {
+	0% {
+		opacity: 1;
+		transform: scale(1);
+	}
+	50% {
+		opacity: 1;
+		transform: scale(1);
+	}
+	100% {
+		opacity: 0;
+		transform: scale(0.9);
+	}
+}
+
+.components-drop-zone__fade-enter {
```

Comment:
There's another downside of doing it with keyframes rather than with a simple `transition`: if the user aborts the animation and quickly restarts it (they drag out of the drop zone and drag into it again), the animation will be reset with `@keyframes`, while with `transition` the browser will start it from where it left off (which was the case with framer motion's implementation IIUC), which will both feel better as a UX and be less work for the browser.

Reply:
I didn't notice this in my tests, but maybe that's because the current animation is very fast. Good to know 👍🏻